web-api/src/persistence/dynamo/deployTable/getStoredApplicationHealth.ts

```diff
@@ -0,0 +1,16 @@
+import { StoredApplicationHealth } from './setStoredApplicationHealth';
+import { getFromDeployTable } from '../../dynamodbClientService';
+
+export const getStoredApplicationHealth = async (
+  applicationContext: IApplicationContext,
+  region: string,
+): Promise<StoredApplicationHealth> => {
+  const result = await getFromDeployTable({
+    Key: {
+      pk: 'healthCheckValue',
+      sk: `healthCheckValue|${region}`,
+    },
+    applicationContext,
+  });
```

Comment:
While I understand the use of the deploy table because it's not application data, I do wonder if the deploy table is the appropriate place to be storing this information. It makes me wonder a broader question about its utility. Couldn't we be storing all information in the main table?

Critically, for this failover story, the Deploy table is not replicated across regions. If Dynamo east were to fail, both regions would be unhealthy.

Reply:
This makes sense and currently we are vulnerable to the deploy table going down. Right now our application as a whole and the failover mechanism is vulnerable to dynamo deploy table going down so we have not made the system as a whole more vulnerable but it is something we should be conscience of. We are adding this note to the follow on SPIKE.